src/components/IncidentTip/IncidentTip.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { PureComponent, PropTypes } from 'react';
+import ReactTooltip from 'react-tooltip';
+
+import { Icon } from '../../components';
+
+import { helpTipContent } from '../../constants';
+
+import '../HelpTip/HelpTip.scss';
+
+/**
+ * Component for (?) tooltips seen around the site.
+ * Uses font awesome's question-circle to render the (?)
+ */
+export default class HelpTip extends PureComponent {
+  static propTypes = {
+    content: PropTypes.string,
+    id: PropTypes.string,
+    place: PropTypes.string,
+    style: PropTypes.string,
+  }
+
+  static defaultProps = {
+    place: 'top',
+    style: 'dark',
+  }
+
+  render() {
+    const { id, style, place } = this.props;
+    let { content } = this.props;
+    if (!content) {
+      content = helpTipContent[id];
+    }
+    const offset = { top: -5, left: -8 };
+
+    return (
+      <span className="HelpTip">
+        <a data-tip data-for={id}><Icon name="exclamation-circle" id={id} /></a>
+        <ReactTooltip
+          id={id}
+          place={place}
+          type={style}
+          effect="solid"
+          offset={offset}
+        >
+          <span dangerouslySetInnerHTML={{ __html: content }} />
```

Comment:
Since this method is also being used in HelpTip (and potentially other components), I am going to make another issue to fix this in a seperate pull request. Currently tracking this at #21